### PR TITLE
ci: fix upload for non org

### PR DESCRIPTION
**Pull request by DelightReza**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -139,6 +139,14 @@ if [[ "${CIRCLE_PROJECT_USERNAME}" == "PitchBlackRecoveryProject" ]]; then
     elif [[ $PB_OFFICIAL == 'true' ]]; then
         ./vendor/utils/pb_deploy.sh OFFICIAL $VENDOR $CODENAME
     fi
+else
+export BUILDFILE=$(find $(pwd)/out/target/product/${CODENAME}/PBRP*.zip 2>/dev/null)
```
Comment (rokibhasansagar) [inline build.sh]: Add Odin tar file support too. Adapt fom this reference: https://github.com/PitchBlackRecoveryProject/vendor_utils/blob/e11d555b2c61a1a7cb48cd21fdaa33356266b0ff/pb_deploy.sh#L214-L219

**Review by rokibhasansagar — COMMENTED** [review]
Rest code will work just fine.

**Comment by AndroiableDroid** [comment]
At L134 if just add  `CIRCLECI=true` condition that will be enough I think

**Comment by manjotsidhu** [comment]
@AndroiableDroid don't you think it should be under deploy script ?

**Comment by AndroiableDroid** [comment]
> @AndroiableDroid don't you think it should be under deploy script ?

nope because deploy is designed as well that it takes the circleci variables without comaprision with ours. so here is the condition comes

**Comment by manjotsidhu** [comment]
> > @AndroiableDroid don't you think it should be under deploy script ?
> 
> nope because deploy is designed as well that it takes the circleci variables without comaprision with ours. so here is the condition comes

Ok

**Review by manjotsidhu — COMMENTED** [review]
LGTM

**Comment by DelightReza** [comment]
> At L134 if just add `CIRCLECI=true` condition that will be enough I think

For what 🙄

**Comment by AndroiableDroid** [comment]
> > At L134 if just add `CIRCLECI=true` condition that will be enough I think
> 
> For what 🙄

revert these changes and replace https://github.com/PitchBlackRecoveryProject/vendor_utils/pull/29/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R134 line with `if [ "${CIRCLECI}" == "true" ]` your issue should be fixed

**Comment by rokibhasansagar** [comment]
> At L134 if just add `CIRCLECI=true` condition that will be enough I think

@AndroiableDroid This will also fail for Non-ORG public as both GITHUB_TOKEN and BOT_API is required by the deploy script.
https://github.com/PitchBlackRecoveryProject/vendor_utils/blob/pb/pb_deploy.sh#L45-L48

And the initial snippet that Raza and I assumed at first also need to reconfigure to work side-by-side with the deploy script (as __build.sh__ is for all public but __pb_deploy.sh__ is for ORG use only).

**Comment by AndroiableDroid** [comment]
@rokibhasansagar I didn't know about it. But if they building from other repo + other account then it should be relay on config.yml don't you think 🤔

**Pull request closed without merging**